### PR TITLE
DFC-13318 -Updated the Add-on with field changes

### PR DESCRIPTION
**Pull request by vallabha24**

### CODE REVIEW

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Export/JobProfile/Dynamic modules/Job Profile/Structure/Job Profile.sf [hunk 1]
```diff
@@ -6973,11 +6912,42 @@
...
+          "TypeUIName": "\n                                Yes / No\n                            ",
+          "Id": "e89d7654-2e60-4539-80cc-bebe5d557f8d",
+          "LastModified": "2020-10-30T09:16:27.243Z"
+        },
         {
           "ParentTypeId": "63177677-3adb-4cb7-8bd2-136e4c34201f",
           "FieldNamespace": "Telerik.Sitefinity.DynamicTypes.Model.JobProfile.Skill",
           "Name": "PSFNotApplicable",
-          "Title": "PSFNotApplicable",
+          "Title": "    PSFNotApplicable",
```
Comment (jbarnes16503) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Export/JobProfile/Dynamic modules/Job Profile/Structure/Job Profile.sf:6950]: This intentional?

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Export/JobProfile/Dynamic modules/Job Profile/Structure/Job Profile.sf [hunk 2]
```diff
@@ -6973,11 +6912,42 @@
           "Id": "e839a46c-db69-49ad-abea-698e8ba258be",
           "LastModified": "2020-10-29T15:48:42.667Z"
         },
+        {
+          "ParentTypeId": "63177677-3adb-4cb7-8bd2-136e4c34201f",
+          "FieldNamespace": "Telerik.Sitefinity.DynamicTypes.Model.JobProfile.Skill",
+          "Name": "PSFHidden",
+          "Title": "    PSFHidden",
```
Comment (jbarnes16503) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Export/JobProfile/Dynamic modules/Job Profile/Structure/Job Profile.sf:6919]: We have whitespace here again...

**Review by jbarnes16503 — CHANGES_REQUESTED** [review]
Couple comments

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Export/JobProfile/Dynamic modules/Job Profile/Structure/Job Profile.sf (hunk 1 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Export/JobProfile/Dynamic modules/Job Profile/Structure/Job Profile.sf:6950]: yep just to add a gap between title and the checkbox. Doesn't effect the code

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Export/JobProfile/Dynamic modules/Job Profile/Structure/Job Profile.sf (hunk 2 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Export/JobProfile/Dynamic modules/Job Profile/Structure/Job Profile.sf:6919]: yep just to add a gap between title and the checkbox. Doesn't effect the code

**Review by jbarnes16503 — APPROVED** [review]
Looks fine

**Pull request merged**